python-avd/tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from pathlib import Path
+from sys import path
+
+path.insert(0, str(Path(__file__).parents[1]))
```

Comment:
why is this required?

Reply:
running pytest from IDE will not start it from the "python-avd" folder so to ensure you pick up the correct code.

Reply:
maybe a little comment on top of it? :)

Reply:
AH  my diff was outdated... resolving